### PR TITLE
[DNM] minified js

### PR DESCRIPTION
**Pull request by lhoffbeck**
## Description

Fixes (issue #)

## Type of change

- [ ]  Patch: Bug (non-breaking change which fixes an issue)
- [ ]  Minor: New feature (non-breaking change which adds functionality)
- [ ]  Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] I have added a changelog entry, prefixed by the type of change noted above (Documentation fix and Test update does not need a changelog as we do not publish new version)

### CODE REVIEW

**Comment by lhoffbeck** [comment]
@atesgoral updated to json.parse, without it we'd have to skip linting anyway so we might as well :)

**Comment by GoodForOneFare** [comment]
tl;dr on the JSON parse thing:

> Because the JSON grammar is much simpler than JavaScript’s grammar, JSON can be parsed more efficiently than JavaScript.

Source: https://v8.dev/blog/cost-of-javascript-2019#json

**Comment by lhoffbeck** [comment]
@GoodForOneFare that's WILD. TIL.

**Comment by atesgoral** [comment]
Some unscientific benchmarking via:

```
const {performance} = require('perf_hooks');

(async () => {
  const start = performance.now();
  const startHeap = process.memoryUsage().heapUsed;

  const data = await import(`./${process.argv[2]}.mjs`);

  const end = performance.now();
  const endHeap = process.memoryUsage().heapUsed;

  console.log(`Elapsed: ${end - start} ms`);
  console.log(`Heap: ${endHeap} - ${startHeap} = ${endHeap - startHeap} B`);;
})();

```

original:
Elapsed: 15.232870995998383 ms
Heap: 3747472 - 2764688 = 982784 B
Elapsed: 13.00815699994564 ms
Heap: 3750824 - 2742896 = 1007928 B
Elapsed: 13.219125002622604 ms
Heap: 3747608 - 2742896 = 1004712 B

minified:
Elapsed: 13.069517999887466 ms
Heap: 3439200 - 2742896 = 696304 B
Elapsed: 11.20490299910307 ms
Heap: 3444352 - 2744416 = 699936 B
Elapsed: 11.85274899750948 ms
Heap: 3440928 - 2764688 = 676240 B

JSON:
Elapsed: 13.069833002984524 ms
Heap: 3784576 - 2742896 = 1041680 B
Elapsed: 11.916978999972343 ms
Heap: 3787600 - 2759784 = 1027816 B
Elapsed: 10.402779996395111 ms
Heap: 3789384 - 2743944 = 1045440 B

Minified seems better?

File: packages/address-mocks/src/fixtures/countries_en.ts [hunk 1]
```diff
@@ -1,11437 +1,4 @@
-const data = {
```
Comment (BPScott) [inline packages/address-mocks/src/fixtures/countries_en.ts:1]: To keep the nicer formating and make this easier to read and update in the future we could do:

```
//prettier-ignore
const data = JSON.parse(`{
  data: BLAH
}`)

**Comment by dahukish** [comment]
@lhoffbeck can we convert this to Draft instead of DNM? Cheers.

**Comment by lhoffbeck** [comment]
@dahukish since it sounds like we want to update how we're generating these altogether, might as well just close this out. Thanks for your help everyone!

**Pull request closed without merging**